libbeat/docs/security/users.asciidoc

```diff
@@ -0,0 +1,295 @@
+[role="xpack"]
+[[feature-roles]]
+=== Grant users access to secured resources
+
+//TODO: Redirect beats-user-access to feature-roles to prevent 404s.
+
+// REVIEWERS: Our built-in roles don't provide enough granularity for some users,
+// and in some cases, the roles don't cover all the tasks that a single user
+// might perform (such as running the setup). Plus the roles don't grant access
+// to Beats indices. I list privileges here (rather than roles) so that the person
+// setting up security has more flexibility and the knowledge they need to
+// create their own roles, if necessary. Is this a likely scenario?
+
+You can use role-based access control to grant {beatname_uc} users access to
+secured resources. The roles that you set up depend on your organization's
+security requirements and the minimum privileges required to use specific
+features. 
+
+{beatname_uc} users typically perform these main roles: they do the initial
+setup, publish monitoring information, and publish events. If they're using
+{kib}, they view and sometimes create visualizations that access {beatname_uc}
+indices.
+
+To assign the most restrictive privileges to users, create a role for each
+purpose. The easiest way to create roles is by using the
+*Management > Security > Roles* UI in {kib}.
+
+Also consider using the <<built-in-beats-roles,built-in roles>> provided by
+{security}.
```

Comment:
Originally the builtin beats role was supposed to be used with xpack montirong only. Not sure if it's a good idea to reuse them.

Reply:
I think the link text here is a little misleading. The section that I'm pointing to describes all the built-in roles that are relevant to beats.